src/streamlink/plugins/openrectv.py

```diff
@@ -149,6 +149,7 @@ def _get_streams(self):
                 log.error("There is no video file.")
 
             if m3u8_file is not None:
+                self.session.http.headers.update({"Referer": "https://www.openrec.tv/"})
```

Comment:
Probably better to use `self.url` instead, so the plugin doesn't rely on a hardcoded string literal.

And as an alternative to setting the header on the entire http session, `headers={...}` could be set as a keyword on the `parse_variant_playlist()` call.